xmind/tests/test_topic_element.py

```diff
@@ -469,3 +469,377 @@ def test_getMarkers(self):
         _marker_fefs_element.getChildNodesByTagName.assert_called_once_with(
             TAG_MARKERREF)
         self._assert_init_methods()
+
+    def test_addMarker_markerId_is_none(self):
+        _element = TopicElement()
+
+        _get_markerrefs = patch.object(_element, '_get_markerrefs').start()
+
+        self.assertIsNone(_element.addMarker(None))
+        _get_markerrefs.assert_not_called()
+        self._assert_init_methods()
+
+    def test_addMarker_markerId_is_str(self):
+        _element = TopicElement()
+
+        _get_markerrefs = patch.object(_element, '_get_markerrefs').start()
+        _get_markerrefs.side_effect = Exception
+
+        _marker_refs_element_constructor_mock = self._init_patch_with_name(
+            '_marker_refs_element_constructor_mock',
+            'xmind.core.topic.MarkerRefsElement'
+        )
+
+        _marker_id_constructor = self._init_patch_with_name(
+            '_marker_id_constructor',
+            'xmind.core.topic.MarkerId',
+            return_value='new_marker_id'
+        )
+
+        with self.assertRaises(Exception) as _ex_mock:
```

Comment:
_ex_mock is not used. Maby it could be  _get_markerrefs.side_effect = Exception('test exception') and then _ex_mock could be checked for what exception was caught.

Reply:
Addressed